hefquin-vocabs/feddesc.ttl

```diff
@@ -0,0 +1,90 @@
+@prefix rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl:    <http://www.w3.org/2002/07/owl#> .
+@prefix xsd:    <http://www.w3.org/2001/XMLSchema#> .
+@prefix dc:     <http://purl.org/dc/elements/1.1/> .
+@prefix fd:     <http://w3id.org/hefquin/feddesc#> .
+
+<http://w3id.org/hefquin/feddesc>
+    rdf:type owl:Ontology ;
+    dc:creator       "Olaf Hartig",
+                     "Robin Keskisärkkä" ;
+    dc:date          "2025-05-05" ;
+    dc:format        "RDF" ;
+    dc:identifier    <http://w3id.org/hefquin/feddesc> ;
+    dc:title         "Federation Description Vocabulary for HeFQUIN"@en ;
+    rdfs:label       "Federation Description Vocabulary for HeFQUIN"@en ;
+    dc:description   "An RDF vocabulary for describing members of a federation to be considered for querying by the HeFQUIN engine."@en ;
+    rdfs:comment     "An RDF vocabulary for describing members of a federation to be considered for querying by the HeFQUIN engine."@en .
+
+### ===== CLASS DECLARATIONS =====
+
+fd:FederationMember a owl:Class ;
+    rdfs:label "Federation Member"@en ;
+    rdfs:comment "An entity that is part of the federation and provides data via a specific interface."@en .
+
+fd:EndpointInterface a owl:Class ;
+    rdfs:label "Endpoint Interface"@en ;
+    rdfs:comment "An interface to access a federation member."@en .
```

Comment:
Please extend this comment in a way similar to how my other review comment proposes to change the comment for `fd:FragmentInterface`.